### ISSUE TITLE
Error log too noisy?

### ISSUE THREAD

**Issue by nevillelyh** [spotify/scio-idea-plugin#2]
Getting the following errors quite often, especially right after I enter some `BigQueryType.fromQuery` macro. Maybe change the plugin to be less noisy or use a lower log level?

```
Throwable
Scio plugin could not find scala files for code completion. Please (re)compile the project.
Missing: /var/folders/hd/f_wgpmsj3h1cv071xszfmyzh0000gn/T//bigquery-classes/Track-8c04c79d.scala
```

**Comment by ravwojdyla** [comment]
Yea - you are right. Was trying to find an easy way to:
 * not alert too much
 * still inform user that she needs to recompile
but I could not find a way to control how often the plugin gets called - it's all Hollywood. I feel like we will have to hack this around, the more I think about this code, the more I feel I want to take a 🚿

**Comment by ravwojdyla** [comment]
[This](https://github.com/JetBrains/intellij-scala/blob/idea15.x/src/org/jetbrains/plugins/scala/lang/psi/impl/toplevel/typedef/SyntheticMembersInjector.scala#L86) is the piece of code that calls the injectors, and it's getting called by [this](https://github.com/JetBrains/intellij-scala/blob/32dd1c48e5de3f1814b7493e8b0a83e81d9c2956/src/org/jetbrains/plugins/scala/lang/psi/api/toplevel/typedef/ScTypeDefinition.scala#L73) and then I think `processScala`. There is a lot of code. If you can think of an easier way to fix this than #3 . Please let me know.
